src/editor/mod.rs

```diff
@@ -142,6 +142,14 @@ impl Editor {
                 trace!("Cursor on");
                 self.cursor.enabled = true;
             }
+            RedrawEvent::Bell => {
+                tracy_zone!("EditorBell");
+                let _ = self.event_loop_proxy.send_event(WindowCommand::Bell.into());
+            }
+            RedrawEvent::VisualBell => {
+                tracy_zone!("EditorVisualBell");
+                // TODO: Implement visual bell
```

Comment:
Do you need help implementing this?

Reply:
Yes, I am not familiar with skia...

Reply:
I think you can add `draw_rect`  (https://rust-skia.github.io/doc/skia_safe/canvas/struct.Canvas.html#method.draw_rect) to `renderer::draw_frame` after everything else is rendered. Either black or white, with some alpha that animates in and out. Compare against the default background color, and select using this condition 
https://github.com/neovide/neovide/blob/4096e4c3465495ed7bd656edf8fd6db28df9c386/src/editor/mod.rs#L595-L597

The updating of the alpha value could be done in `renderer::animate_frame`